### DISCUSSION TITLE
国区年付许可

### DISCUSSION THREAD

**Discussion by izhangzhihao** [izhangzhihao/intellij-rainbow-brackets#2667] [General]
想要永久许可？请参考[这里](https://github.com/izhangzhihao/intellij-rainbow-brackets/discussions/2587)

最新折扣和使用手册请参考：https://docs.qq.com/doc/DQ3NaT1NJVUJka2h0

**Comment by HipolitoGallaty** [comment]
大概多久会给发license。

**Reply by izhangzhihao** [reply]
白天通常十分钟内会发，最长不会超过一小时。

**Reply by everstu** [reply]
![PixPin_2024-02-29_15-54-19](https://github.com/izhangzhihao/intellij-rainbow-brackets/assets/19686947/3513f2cf-9e32-499b-8ba4-7022c9773c51)
这个限制是为了防止插件破解吗？

**Reply by izhangzhihao** [reply]
是的，IDE 如果被破解了，插件基本也就被破解了。

**Reply by XuBoZhangQvQ** [reply]
有个问题，如果我购买了license，但是我用了ja-netfiltter ，是不是依旧会提示这个，激活不了？
我就想支持一下咱们插件

**Reply by everstu** [reply]
yes，只要用ja-netfiltter破解就不能激活，作者给的建议是用社区免费版的IDE。

**Reply by XuBoZhangQvQ** [reply]
害，已经用了好久的idea了，切到社区版ide成本太大了，还好我没下单，要不就算下了单，还是一样不可用....

**Comment by OrangeCat3** [comment]
现在买的还没过期呢，

**Reply by izhangzhihao** [reply]
没听懂你的问题，如果你现在还有未过期的 license，新的 license和旧的是可以叠加的，时间会在原来的基础上延长。

**Reply by OrangeCat3** [reply]
好的

**Comment by chen09** [comment]
"仅支持中国区账户激活使用"是什么意思？我买的某宝的应该是海外某校的学生账号，就不可以激活了吗？

**Reply by izhangzhihao** [reply]
实际上其他 region 的账号也可以激活，我发现很多是海外华人，就放开了限制。

**Reply by chen09** [reply]
明白了，我的确也在海外。这样就放心了。支持一下国内优秀plugin创作者

**Reply by chen09** [reply]
再问一个问题，折扣代码是通过，支付宝给我的吧？（我准备用支付宝支付）

**Reply by izhangzhihao** [reply]
你留言一个邮箱，我通过邮件发给你～

**Reply by chen09** [reply]
已支付。chen09@gmail.com 谢谢

**Reply by izhangzhihao** [reply]
已发送，如果没收到还请检查下垃圾箱。

**Comment by chen09** [comment]
> 已发送，如果没收到还请检查下垃圾箱。

cool, it is worded. 三克油

**Comment by JinheJamesLi** [comment]
你给我的折扣码是一年的吗？如果是一年的折扣，我可以获得这一年内永久的订阅版本吗？

**Reply by izhangzhihao** [reply]
如果你指的是[fallback license](https://sales.jetbrains.com/hc/en-gb/articles/207240845-What-is-a-perpetual-fallback-license-)的话，这种情况不适用于这个插件。

**Comment by ywhlhy** [comment]
请问如果到时候jetbrains账号被封了，可以在新账号上继续应用这个已经购买的折扣码？

**Reply by izhangzhihao** [reply]
Ah, 如果账号被封了可以邮件我详细信息，我们看具体情况

**Comment by ZzhangYH** [comment]
已支付，邮箱已在支付宝付款时备注（299***936@qq.com），感谢！

**Reply by izhangzhihao** [reply]
已发送，请查收，如果没收到请检查下垃圾收件箱

**Comment by twins49** [comment]
已支付宝支付，邮箱：77135016@qq.com

**Reply by izhangzhihao** [reply]
已发送，请检查收件箱或者垃圾箱

**Comment by davichi11** [comment]
已支付 微信,邮箱:davinci1987@icloud

**Reply by davichi11** [reply]
davinci1987@icloud.com

**Reply by izhangzhihao** [reply]
已发送，请查收

**Comment by hi-asan** [comment]
微信已经支付了，并且备注过邮箱地址，需要怎么邮箱联系您

**Reply by izhangzhihao** [reply]
已发送，请查收
